tuxemon/monster.py

```diff
@@ -384,6 +384,7 @@ def apply_status(self, status: Condition) -> None:
         count_status = len(self.status)
         if count_status == 0:
             self.status.append(status)
+            print("piss", self.name)
```

Comment:
yeah, just testing

Reply:
I know it was for testing, but since you opened your PR for reviews, I assumed you were not testing it anymore (hence that you just forgot to remove the testing statements).